EleksTubeHAX_pio/src/_USER_DEFINES - empty.h

```diff
@@ -32,6 +32,7 @@
 #define DAY_TIME                    7   // full brightness after 7 am
 #define BACKLIGHT_DIMMED_INTENSITY  1   // 0..7
 #define TFT_DIMMED_INTENSITY        20  // 0..255
+//#define DIM_WITH_ENABLE_PIN_PWM         // uncomment to enable hardware dimming with TFT Enable pin (GPIO 4)
```

Comment:
This can easily return in problems if users enable it for hardware that does not support it.

I would not add an extra define and use the existing `HARDWARE_IPSTUBE_CLOCK` since only that hardware supports backlight dimming via pwm for now, then it's also already enabled for users that use IPSTube.

Reply:
Are you sure that every `HARDWARE_IPSTUBE_CLOCK` clock supports such feature?
Because I am not sure whether all such clocks actually have Q1 transistor and support that (enabling the feature in such case will make both software and hardware dimming inoperable).

Reply:
> Are you sure that every `HARDWARE_IPSTUBE_CLOCK` clock supports such feature? Because I am not sure whether all such clocks actually have Q1 transistor and support that (enabling the feature in such case will make both software and hardware dimming inoperable).

Since they all have gpio4 exposed to that transistor it's fine.

My H401 also has Q1 soldered.